consensus/consensus-types/src/block_retrieval.rs

```diff
@@ -9,20 +9,36 @@ use serde::{Deserialize, Serialize};
 use short_hex_str::AsShortHexStr;
 use std::fmt;
 
-pub const MAX_BLOCKS_PER_REQUEST: u64 = 10;
+// this number should be no smaller than backpressure limit
```

Comment:
why this can't be smaller than back pressure limit since we support multi chunks?

Reply:
Oh I mean `recommended to be no smaller`